docs/source/concept_models/vivarium_ciff_sam_2/concept_model.rst

```diff
@@ -78,11 +84,11 @@ The goal of this simulation is to investigate the question:
     - Six months
     - 
   * - Age end
-    - 2 years
-    - Consider increasing to 5?
+    - 5 years
+    - 
   * - Exit age
-    - 2 years
-    - Consider increasing to 5?
+    - 5 years
+    - 
   * - Simulation start date
     - 2021-07-01
```

Comment:
Let's shift everything forward a year, so we start in 2022, and start observation in 2023
```suggestion
    - 2022-07-01
```

Reply:
Ok!

Reply:
Actually, when I went to make these changes, I realized this would cause us to have to redefine the scale-up algorithm we used in the first phase of CIFF. Does this seem worth it when the dates are arbitrary? I'd prefer that we will just report "year 1, 2, 3, etc." in the paper to avoid the implication that we're using year-specific forecasts.